Formula/gdal.rb

```diff
@@ -59,6 +58,7 @@ class Gdal < Formula
   depends_on "python@3.11"
   depends_on "qhull"
   depends_on "sqlite"
+  depends_on "swig"
```

Comment:
This is probably a `:build` dependency.

Please also squash your commits together so you only have one commit.

Thanks!

Reply:
Thanks, I'll fix that.